src/native/libs/System.Security.Cryptography.Native/pal_ssl.c

```diff
@@ -586,6 +599,61 @@ void CryptoNative_SslCtxSetCaching(SSL_CTX* ctx, int mode)
     {
         SSL_CTX_set_options(ctx, SSL_OP_NO_TICKET);
     }
+
+    if (newSessionCb != NULL || removeSessionCb != NULL)
+    if (newSessionCb != NULL)
```

Comment:
This logically nested (but not indented) if is really just the latter condition.  One of these two lines should be deleted.

Reply:
removed 603. I probably started something I end up not needing.